tests/integration/test_end_to_end_dcp.py

```diff
@@ -163,5 +163,73 @@ def test_10x_run(self):
             raise TimeoutError("test timed out")
 
 
+class TestOptimusRun(TestEndToEndDCP):
```

Comment:
So we are not removing the cellranger runner, instead, we don't run it?

Reply:
I'll actually remove the test for cellranger as well just so there isn't any confusion there